appveyor.yml

```diff
@@ -11,13 +11,19 @@ before_build:
 
 platform: Any CPU
 
+environment:
+  _CL_: /c /analyze /analyze:quiet- /W4 /analyze:autolog:ext prefast.sarif
+
 configuration:
 #  - Debug
   - Release
 
 build:
   project: src/Everything.sln
-  
+
+before_test:
```

Comment:
Agreed, I meant to mention that. The important thing the code needs to convey here is "you can't do this until the build is done". As written, the code conveys "you can't run the tests before you do this" -- which isn't true.